docs/ADRs/developmentLanguage.md

```diff
@@ -0,0 +1,47 @@
+# Title
+
+Development language
+
+## Status
+
+Accepted June 2020
```

Comment:
This is an example date - it is ok for ADRs to be revised at a later date - sometimes it is good to know when a decision was made because technology can be "of a time" rather than forever.
I made this date up. Please correct it if you have a better one :)